src/components/Config/Config.tsx

```diff
@@ -120,68 +146,148 @@ function Config() {
     }
   }, [checkConnection, wallet]);
 
+  const ArbitrumNetworkConnect = useCallback(async () => {
+    const ArbitrumTestChainId = '0x66eeb'; // arbitrum test net
+    if (typeof window.ethereum !== 'undefined') {
+      const chainId = await window.ethereum.request({ method: `eth_chainId` });
+      if (chainId === ArbitrumTestChainId) {
+        notification.success({ message: `You're connected to Automate!` });
+      } else {
+        console.log('oulalal, switch to the correct network,chainid: ' + chainId);
+        try {
+          await window.ethereum.request({
+            method: 'wallet_switchEthereumChain',
+            params: [{ chainId: ArbitrumTestChainId }],
+          });
+          console.log('You have succefully switched to Arbitrum Test network');
+        } catch (chainerror) {
+          if (chainerror.code === 4902) {
+            console.log('This network is not available in your metamask, please add it');
+          }
+          try {
+            await window.ethereum.request({
+              method: 'wallet_addEthereumChain',
+              params: [
```

Comment:
This will have to be changed so that the rpc url is dynamically built based on the user inputs. Also the block explorer url should come from `constants.ts` at least. Maybe this while config object should just be moved there and then only the rpc url updated on the fly